src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -1316,6 +1332,15 @@ void SolidMechanicsLagrangianFEM::resetStateToBeginningOfStep( DomainPartition &
         incdisp( a, i ) = 0.0;
       }
     } );
+
+    ElementRegionManager & elementRegionManager = mesh.getElemManager();
+    elementRegionManager.forElementSubRegions< CellElementSubRegion >( regionNames,
+                                                                       [&]( localIndex const,
+                                                                            CellElementSubRegion & subRegion )
+    {
+      solidMechanics::arrayView2dLayoutIncrStrain incStrain = subRegion.getField< solidMechanics::incrementalStrain >();
+      incStrain.zero();
```

Comment:
```suggestion
 subRegion.getField< solidMechanics::incrementalStrain >().zero();
```

Reply:
if you are setting it to zero here I am not sure I see the point of that `if (q == 0)`. I also think that this means that the field will always be zero when we plot it so I would only store the strain and not the incremental one.